helm/fluent-logshipping-app/templates/priorityClass.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.fluentbit.customEnabled }}
```

customEnabled? Can you maybe use something like 
```suggestion
{{- if .Values.fluentbit.priorityClass.create }}
```

---

Sure, makes more sense